setup.py

```diff
@@ -213,34 +209,43 @@ def check_file_at_path(path2file):
         else:
             omplinkflags = ['-fopenmp']
 
-    if tbb_root:
-        print("Using Intel TBB from:", tbb_root)
-        ext_np_ufunc_tbb_backend = Extension(
-            name='numba.np.ufunc.tbbpool',
-            sources=[
-                'numba/np/ufunc/tbbpool.cpp',
-                'numba/np/ufunc/gufunc_scheduler.cpp',
-            ],
-            depends=['numba/np/ufunc/workqueue.h'],
-            include_dirs=[os.path.join(tbb_root, 'include')],
-            extra_compile_args=cpp11flags,
-            libraries=['tbb'],  # TODO: if --debug or -g, use 'tbb_debug'
-            library_dirs=[
-                # for Linux
-                os.path.join(tbb_root, 'lib', 'intel64', 'gcc4.4'),
-                # for MacOS
-                os.path.join(tbb_root, 'lib'),
-                # for Windows
-                os.path.join(tbb_root, 'lib', 'intel64', 'vc_mt'),
-            ],
-        )
-        ext_np_ufunc_backends.append(ext_np_ufunc_tbb_backend)
+    # Disable tbb if forced by user with NUMBA_NO_TBB=1
+    if os.getenv("NUMBA_NO_TBB") == "1":
```

Comment:
Need it be "1", won't anything True-ish work ok?

Reply:
It is a string so any non empty string would work. 
I think keeping it to `"1"` makes it easy to keep track of.
Stylistic choice, any change is fine.

Reply:
I would prefer to keep it as a non-empty string. This is the interface we have been using for `NUMBA_NO_OPENMP` and constraining it to be explicitly `1` would alter that interface, violate an existing contract we have and potentially break other peoples builds. So, in that light, I think it would be a good idea to remain consistent as to how we handle environment variables including the newly introduced `NUMBA_NO_TBB`.